src/components/ImageUploader/src/ImagePicker.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+	<div
+		:class="$s.ImagePickerInputContainer"
+		@click="$refs.input.click()"
+	>
+		<input
+			ref="input"
+			:class="$s.ImagePickerInput"
+			type="file"
+			multiple="true"
+			accept="image/jpeg, image/png"
```

Comment:
I'm thinking this should be replaced with an `accept` prop that defaults to `image/*`